x-pack/plugins/observability/public/pages/overview/overview.stories.tsx

```diff
@@ -37,6 +37,7 @@ const sampleAPMIndices = { transaction: 'apm-*' } as ApmIndicesConfig;
 const withCore = makeDecorator({
   name: 'withCore',
   parameterName: 'core',
+  // @ts-expect-error update types
```

Comment:
Is this planned for another PR? should our team take care of it?

Reply:
hey @afgomez, it turned out that isn't necessary anymore